services/user.py

```diff
@@ -0,0 +1,48 @@
+from typing import Optional
+
+from db.models import User
+
+
+def create_user(
+        username: str,
+        password: str,
+        email: Optional[str] = None,
+        first_name: Optional[str] = None,
+        last_name: Optional[str] = None,
+) -> None:
+    User.objects.create_user(
```

Comment:
![image](https://user-images.githubusercontent.com/104974881/224480626-97a6d19a-676e-4c74-9f3a-ccc4f351b436.png)
use here and other places